Utils/StringUtils.cs

```diff
@@ -20,6 +20,7 @@ public static IEnumerable<HtmlNode> TextDescendants(HtmlNode root)
 
     public static string TruncateHtml(string html, int maxCharacters)
     {
+      html = html.Replace("<p>", "").Replace("</p>", "");
```

Comment:
The result is already added in a `<p class="latest_copy">` tag. Nested paragraphs isn't valid. w3 validator was complaining for that.